examples/conversation_sql/database/concert_singer/schema.sql

```diff
@@ -0,0 +1,85 @@
+PRAGMA foreign_keys = ON;
```

Comment:
What's the usage of this file?

Reply:
Now use the schema.sql to create the .sqlite file.